app/views/shared/_orderlist.html.erb

```diff
@@ -22,55 +22,83 @@
     <tr>
       <% if subscription.status == "pending" %>
```

Comment:
'pending' needs to be capital 'Pending'. In the database, the status is set with capital letter.